posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -212,7 +212,7 @@
       team_id Int64,
       properties VARCHAR,
       is_identified Int8,
-      is_deleted Int8 DEFAULT 0,
```

Comment:
`Int8` defaults to `0` here without the `DEFAULT` statement and the default statement breaks the kafka engine on `23.4`

Reply:
we can also explicitly write that if that would be helpful